workers/unity/Packages/io.improbable.gdk.core/Utility/Snapshot.cs

```diff
@@ -39,6 +44,16 @@ public EntityId AddEntity(EntityTemplate entityTemplate)
             return entityId;
         }
 
+        /// <summary>
+        ///     Checks if an entityId is valid
+        /// </summary>
+        /// <param name="entityId">The entity ID to check for</param>
+        /// <returns></returns>
+        public bool IsValid(EntityId entityId)
```

Comment:
Nit: Call this `Contains` instead? 

There is an overloaded concept of a valid entity ID